### ISSUE TITLE
All Groups are deleted and incorrect Last Backup date

### ISSUE THREAD

**Issue by Tyrewt** [Drive4ik/simple-tab-groups#287]
Two potential bugs.

1. Randomly, with no error indication I've noticed all Groups are removed. Clicking on the STG icon, all tabs appear to be present, but are unsorted from their respective Groups with the message "**Found hidden unsynchronized tabs. Click on tab to show it into current group/window or:**"

2. Currently daily backup is set, so going to restore from backup. I've noticed the last backup date is reported incorrectly (unless I've invented time travel), is says: **1969-12-31**

![screenshot from 2018-11-12 00-36-50](https://user-images.githubusercontent.com/21961083/48328950-87fc6680-e614-11e8-96e2-20f6209eb1a7.png)

Ubuntu 18.04
Firefox 63.0
Simple Tab Groups 3.3.4

**Comment by kotarou3** [comment]
I've also encountered bug 1 above. Happened last week as well, both on the Tuesday.

Ubuntu 16.04.5
Firefox 63.0
STG 3.3.4

**Comment by Tyrewt** [comment]
> This happened again today, another Tuesday

I do not believe it is date specific. I've experienced Bug #1 again today (Thursday, Nov 22).

**Comment by kotarou3** [comment]
Happened again for me, this time Saturday/Sunday Dec 1/2

**Comment by Despruk** [comment]
Just had this issue yesterday for the first time.

Arch Linux
Firefox 63.0.3
Simple Tab Groups 3.3.4
